.github/workflows/native-tests-daily.yml

```
@@ -22,3 +24,11 @@ jobs:
           gu --version
           native-image --version
           ./gradlew nativeTest
+
+  workflow-notification:
+    needs:
+      - analyze
```

Shouldn't it be `build`?
```suggestion
      - build
```